### PR TITLE
complete nc_news_7

### PR DESCRIPTION
**Pull request by alicjafleszar**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -221,6 +282,56 @@ describe('Error handling tests', () => {
...
+                    .post('/api/articles/3/comments')
+                    .send({
+                        "username": "deletedAccount",
+                        "body": "cool article"
+                    })
+                    .expect(404)
+                    .then(({ body: { msg } }) => {
+                        expect(msg).toBe('Not Found')
+                    })
+            })
```
Comment (johnhutcheon) [inline __tests__/app.test.js:355]: Really nice testing so far Alicja! Could you just add one more 201 test please, which ignores unnecessary properties? So if for example we add something apart from username and body into the object we're sending, it should ignore it and still post. 👍

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Hey Alicja, nice work! Could you just see my comment please and add the extra test? Thanks1

File: __tests__/app.test.js (hunk 1 above)
Comment (alicjafleszar) [reply __tests__/app.test.js:355]: Thanks! Just added the test. Does it have everything it needs?

**Review by johnhutcheon — APPROVED** [review]
All good, thanks for making the change. Merge away! 🙂

**Pull request merged**